manifests/init.pp

```diff
@@ -437,6 +437,59 @@
   Hash               $mutators = {},
   ### END Hiera Lookups ###
 ) {
+
+  case $::osfamily {
```

Comment:
perhaps this should be done based on `$kernel` so as we add Linux distro's, we don't have to keep adding to the list of matching osfamily's.